styling/theme.js

```diff
@@ -1,11 +1,14 @@
 export default {
   black: '#4F463F',
+  whiteAlpha: '#FFFFFF80',
```

Comment:
That is pretty cool I didn't know that there was an option to do this. However, it looks like this is a new feature https://caniuse.com/#feat=css-rrggbbaa. It is not supported by all IE browsers. Probably better to fall back to `RGBA`. 

Or you could define it as a HEX colour code but convert it javascript function to `RGB` and then add an `A`lpha channel on your own.

Reply:
Oh, I thought that was age-old syntax. Will fix.